modules/oas-page-builder/src/services/pageBuilder.ts

```diff
@@ -148,7 +152,40 @@ async function getOASpec({
 
     const path = `${output}/${pageSlug}${filePathExtension}/index.html`;
     const finalFilename = normalizePath(path);
-    await redocExecutor.execute(spec, finalFilename, buildOptions);
+
+    let versionOptions: RedocVersionOptions | undefined;
+
+    if (resourceVersions && apiVersion) {
+      const rootUrl = `${siteUrl}/${pageSlug}`;
+
+      // if there is no resource version provided, but there is a resourceVersions array present,
+      // get the latest resource version from the array, and assign it to the active resource version
+      if (!resourceVersion) {
+        const latestResourceVersion = resourceVersions.sort((a, b) => new Date(a).getTime() - new Date(b).getTime())[
```

Comment:
I did NOT realize how much faster this comparison is than just subtracting two Dates!! That is a wild thing I learned today.